### ISSUE TITLE
Switch email sign-ups to use Exact Target (via Basket)

### ISSUE THREAD

**Issue by adamlofting** [mozilla/donate.mozilla.org#358]
Our mailing lists will be setup on Basket Staging immenently.
http://basket.readthedocs.org/en/latest/

**Comment by ScottDowne** [comment]
I'm OK with using BSD for this. I suggest we make this a P2.

**Comment by adamlofting** [comment]
This might have implications for how we migrate mailing lists to ET (I'm working on the details this week). But if it does I'll take it as my responsibility before launch.

**Comment by adamlofting** [comment]
We've solved for this recently in petitions.m.o https://github.com/mozilla/petitions.mozilla.org/tree/master/lib

**Comment by ScottDowne** [comment]
Ah, it's a P2, which is why I couldn't find it.

Wouldn't this need to be a P1? When I made it a P2 previously, I figured BSD would be fine for Germany. Is that still the case?

**Comment by adamlofting** [comment]
It's a weird P2.

We could ship without any email opt-in (as is the case with currently live de form) which is what made it a P2.

But we can't ship with BSD form because de needs the double opt-in.

**Comment by ScottDowne** [comment]
https://donate.mofostaging.net/de/thank-you/

It's currently using BSD, so I would say yes, it needs to be a P1 now.

Either we remove BSD or add exact target. Removing BSD is an option and the easy solution. I suspect Exact target is doable, but def not as easy as removing BSD.

**Comment by ScottDowne** [comment]
Seemed to be easy enough to implement by copying petitions.

https://github.com/mozilla/donate.mozilla.org/pull/487

However, I cannot seem to get it to send an email test to me locally. It doesn't give me an error.

**Comment by ScottDowne** [comment]
So, in order to try to verify if my patch was working, but I'm just not getting local signups, or if it's busted, I tried petitions locally (which we know works) and I still didn't get an email, so that's a good sign.

I then tried on petitions.mozilla.org, and also didn't get an email... So that was a bit worrying. I have no way to confirm, but the likely explanation is it works, and local and duplicate singups don't send emails?

Putting this on @adamlofting to help me test/review this.
